restorer_test.go

```diff
@@ -47,13 +48,24 @@ func testRestorer(t *testing.T, when spec.G, it spec.S) {
 			testCache, err = cache.NewVolumeCache(cacheDir)
 			h.AssertNil(t, err)
 
+			platform := v06.NewPlatform("0.6")
```

Comment:
```suggestion
			platform := platformPkg.NewPlatform("0.6")
```
(see previous comment)